From 4ddfc589b521af0fbfdf6cb60e3a092294dc5875 Mon Sep 17 00:00:00 2001
From: Yannick Lanz <yannick.lanz@wifx.net>
Date: Tue, 7 Sep 2021 21:13:25 +0200
Subject: [PATCH 1/2] test_loragw_hal_tx: enable CRC for LoRa TX packets by
 default

The RX tool (test_loragw_hal_rx) has been made to count packets with valid CRC.
This patch enables CRC for TX packets by default to make it work.

Signed-off-by: Yannick Lanz <yannick.lanz@wifx.net>
---
 libloragw/tst/test_loragw_hal_tx.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libloragw/tst/test_loragw_hal_tx.c b/libloragw/tst/test_loragw_hal_tx.c
index f9f2757..6f43ba5 100644
--- a/libloragw/tst/test_loragw_hal_tx.c
+++ b/libloragw/tst/test_loragw_hal_tx.c
@@ -520,7 +520,7 @@ int main(int argc, char **argv)
         } else {
             pkt.modulation = MOD_LORA;
             pkt.coderate = CR_LORA_4_5;
-            pkt.no_crc = true;
+            pkt.no_crc = false;
         }
         pkt.invert_pol = invert_pol;
         pkt.preamble = preamble;
-- 
2.25.1

